test/integration/product-import.spec.coffee

```diff
@@ -189,6 +189,36 @@ describe 'Product Importer integration tests', ->
     .then (result) ->
       service.byId(id).delete(result.body.version)
 
+  it 'should handle an empty import', (done) ->
```

Comment:
If you provide an empty array it won't send any unnecessary requests because `Promise.map([], ...)` won't call a processing function here https://github.com/sphereio/sphere-product-import/pull/109/files/054d9a44c61b53e115436a6eaea267980bf5f0bb#diff-d7b70bb31c9da8f845222653763558c7R117